module/reports/helper.js

```diff
@@ -101,63 +103,7 @@ module.exports = class ReportsHelper {
 				projectReport[CONSTANTS.common.SUBMITTED_STATUS] = 0
 				projectReport[CONSTANTS.common.INPROGRESS_STATUS] = 0
 				projectReport[CONSTANTS.common.STARTED] = 0
-
-				let types = await this.types()
-				let returnTypeInfo = types.data.filter((type) => {
-					if (type.value == reportType) {
-						return type.label
-					}
-				})
-				// if (!projectDetails.length > 0) {
-				//Initially not supporting pdf report, hence resolving response here. Remove resolve if pdf capability needs to be added
-				// return resolve({
-				// 	message: CONSTANTS.apiResponses.REPORTS_DATA_NOT_FOUND,
-				// 	data: {
-				// 		dataAvailable: false,
-				// 		data: {
-				// 			categories: categories,
-				// 			tasks: tasksReport,
-				// 			projects: projectReport,
-				// 		},
-				// 	},
-				// })
-
-				if (getPdf == true) {
-					let reportTaskData = {}
-					Object.keys(tasksReport).map((taskData) => {
-						reportTaskData[UTILS.camelCaseToTitleCase(taskData)] = tasksReport[taskData]
-					})
-					let categoryData = {}
-					Object.keys(categories).map((category) => {
-						categoryData[UTILS.camelCaseToTitleCase(category)] = categories[category]
-					})
-					let pdfRequest = {
-						reportType: returnTypeInfo[0].label,
-						sharedBy: userName,
-						reportTitle: pdfReportString,
-						categories: categoryData,
-						tasks: reportTaskData,
-						projects: projectReport,
-					}
-					let response = await common_handler_v2.unnatiEntityReportPdfGeneration(pdfRequest, userId)
-					if (response && response.success) {
-						return resolve({
-							success: true,
-							message: CONSTANTS.apiResponses.REPORT_GENERATED,
-							data: {
-								data: {
-									downloadUrl: response.pdfUrl,
-								},
-							},
-						})
-					} else {
-						return resolve({
-							message: CONSTANTS.apiResponses.REPORTS_DATA_NOT_FOUND,
-							data: [],
-							success: false,
-						})
-					}
-				} else {
+				if (!projectDetails.length > 0) {
```

Comment:
@MallanagoudaB can you please tell me why we removed this Part [line-128 to 185] ref this :
https://github.com/ELEVATE-Project/project-service/pull/23/files#diff-13ab73c9e43ba8f09171958cd34a636841985118315ae882a151e900cd17a009

When enabling report that also needs to be enabled right